From: Jeff Mahoney <jeffm@suse.com>
Subject: [PATCH] iscsi: fix missing semicolon

 Without the attached patch, libiscsi fails to build with debugging enabled.

Signed-off-by: Jeff Mahoney <jeffm@suse.com>
---
 drivers/scsi/libiscsi.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/scsi/libiscsi.c	2007-10-31 17:44:53.000000000 -0400
+++ b/drivers/scsi/libiscsi.c	2007-10-31 17:44:53.000000000 -0400
@@ -1109,7 +1109,7 @@ static int iscsi_exec_abort_task(struct 
 	if (!ctask->mtask) {
 		spin_unlock_bh(&session->lock);
 		iscsi_conn_failure(conn, ISCSI_ERR_CONN_FAILED);
-		spin_lock_bh(&session->lock)
+		spin_lock_bh(&session->lock);
 		debug_scsi("abort sent failure [itt 0x%x]\n", ctask->itt);
 		return -EPERM;
 	}
